init.rb

```diff
@@ -40,8 +41,11 @@
     :possible_values => ['stable', 'master'],
     :doc => [
         "Which flavor of Rock do you want to use ?",
-        "Use 'stable' to use the a released version of Rock that gets updated with bugfixes", "'master' for the development branch","If you want to use a released version of rock, choose 'stable' and then call 'rock-release switch' after the initial bootstrap", "See http://rock-robotics.org/stable/documentation/installation.html for more information"])
+        "Use 'stable' to use the a released version of Rock that gets updated with bugfixes", "'master' for the development branch","Released version of Rock use 'stable'. Use rock-release switch to change from release to flavors.", "See http://rock-robotics.org/stable/documentation/installation.html for more information"])
```

Comment:
"the a" --> a,  i guess  "Released version of Rock use 'stable'" can be removed.

Reply:
:+1: Done.